src/Card/Card.scss

```diff
@@ -5,7 +5,30 @@
 .pgn__card-body {
   width: 100%;
 
-  &.is-muted {
+  &.pgn__card-dark {
```

Comment:
It appears the background color is coming through as if its a border applied to the `Card`: 

<img width="324" alt="image" src="https://user-images.githubusercontent.com/2828721/206717168-1a35bdd6-1e3f-4940-9264-5590884ed9bb.png">

This makes it appear as if it has active/focus styling when it shouldn't, and makes the actual active/focus styles largely indistinguishable, e.g. focus styles on the light variant:

<img width="354" alt="image" src="https://user-images.githubusercontent.com/2828721/206717473-35687f0d-3b86-4472-9511-8714937f397c.png">

Reply:
yeah, it looks like this border is the issue, I removed it for now with `border: 0` but then I'm not sure what the focus styles should be for dark `Card`.. (if we add border only for focus state then it gets jumpy when you focus). Might need to think more about this

Reply:
@viktorrusakov We might consider having the focus state border for `Card` be offset from the card element similar to the focus state on `Button` for all its variants, e.g.:

<img width="345" alt="image" src="https://user-images.githubusercontent.com/2828721/208108728-4c0c1f8e-f4a6-4296-8acb-332f62f908c1.png">

<img width="262" alt="image" src="https://user-images.githubusercontent.com/2828721/208108804-8655296e-0cef-4cfa-a3f5-b1f6c506d49d.png">

Reply:
@viktorrusakov I also believe we'll want to add `border: 0;` for all `Card` variants, not just dark. otherwise the card heights may be slightly inconsistent if rendering different `Card` variants side by side (e.g., light variant next to dark variant).